pages/common/git-cp.md

```diff
@@ -0,0 +1,12 @@
+# git cp
+
+> Copy an existing file. A part of git-extras.
+> More information: <https://github.com/tj/git-extras>.
```

Comment:
```suggestion
> More information: <https://manned.org/git-cp>.
```

Reply:
We tend to use manned.org for the link so I thing this should be used instead

Reply:
Ok I will do this to the other PRs

Reply:
You know, I kinda think we should use https://github.com/tj/git-extras/blob/master/Commands.md#git-cp instead, because, although it's basically indentical to manned.org, it's the "official" page.  Thoughts?

Reply:
no comment

Reply:
I don't mind. I was just trying to apply what I understood before :roll_eyes: :smile:

Reply:
Man page links are for when official documentation either doesn't exist or is not CLI-focused, @patricedenis :)

Reply:
It's not as we if don't keep subtly changing it too...because we're never firmly decided on one this 😆

Reply:
So @CleanMachine1 you may switch back to the link given by @bl-ue  then :smile_cat:

Reply:
163dbb71ffc43e5e2efd0e47dbb722516ea9c682